picard/ui/scripteditor.py

```diff
@@ -260,6 +260,8 @@ class ScriptEditorDialog(PicardDialog):
     TITLE = N_("File naming script editor")
     STYLESHEET_ERROR = OptionsPage.STYLESHEET_ERROR
 
+    help_url = '/config/options_filerenaming_editor.html'
```

Comment:
This is returning a 404 for now, expected?

Reply:
I believe so because it hasn't been merged into the master branch yet.  It's still in the branch collecting the changes for the upcoming 2.7 release.

Reply:
Yes, that's currently expected. We could also link it to the general renaming page for now and update the URL later when we have this page actually available, but I thought we would easily forget about that.